### PR TITLE
Install mimalloc an all images

### PR DESCRIPTION
**Pull request by pvizeli**
We have all images working in the same way just not with the malloc. That fixes it and allow it to be like the other base images.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ Using these images as a base for other Docker projects is, however, not recommen
 
 The image include [S6-Overlay](https://github.com/just-containers/s6-overlay), [Bashio](https://github.com/hassio-addons/bashio) and [TempIO](https://github.com/home-assistant/tempio).
 
+## Mimalloc
+
+We support on our platforms mimalloc. On application which you want enable it, set as envoriement `LD_PRELOAD="/usr/local/lib/libmimalloc.so"` on your Dockerfile or before you start the application.
```
Comment (frenck) [inline README.md:12]:
```suggestion
We support on our platforms mimalloc. On the application which you want to enable it, set as environment `LD_PRELOAD="/usr/local/lib/libmimalloc.so"` on your Dockerfile or before you start the application.
```

File: raspbian/Dockerfile [hunk 2]
```diff
@@ -38,16 +39,30 @@ RUN \
...
+        "https://github.com/microsoft/mimalloc" /usr/src/mimalloc \
+    && mkdir -p /usr/src/mimalloc/build \
+    && cd /usr/src/mimalloc/build \
+    && cmake .. \
+    && make -j "$(nproc)" \
+    && make install \
+    \
     && curl -L -f -s -o /usr/bin/tempio \
-        "https://github.com/home-assistant/tempio/releases/download/${TEMPIO_VERSION}/tempio_armhf" \
+        "https://github.com/home-assistant/tempio/releases/download/${TEMPIO_VERSION}/tempio_${BUILD_ARCH}" \
```
Comment (frenck) [inline raspbian/Dockerfile:51]: `BUILD_ARCH` isn't requested for yet?

Need to be added to the args lists above

Comment (pvizeli) [reply raspbian/Dockerfile]: right can be static like for s6-overlay

Comment (frenck) [reply raspbian/Dockerfile]: Actually, it cannot?

There is a single build file, for multiple architectures in this case. I think this is a bug?
This file is used by amd64 as well.

Comment (pvizeli) [reply raspbian/Dockerfile]: No, we support only amrhf. But I think we should drop that image anyway since debian support armhf

Comment (frenck) [reply raspbian/Dockerfile]: Oh! Right 🤦

**Review by frenck — APPROVED** [review]

**Pull request closed without merging**